tests/core/creators/manual/themeduilabels.md

```diff
@@ -0,0 +1,23 @@
+@bender-tags: bug, 4.19.0, 2445
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, basicstyles, floatingspace
+
+**Note** This test is intended to be used with a screen reader.
+
+1. Focus the editor.
+
+	**Expected** The editor is correctly announced ("Editor hublabubla").
+
+	**Unexpected** The editor is incorrectly announced or not announced at all.
+
+Sample announcements for the `iframe`-based editor:
+
+* VoiceOver: "edit text, &lt;content and selection info&gt;. Editor, wysiwygarea, group"
+* NVDA: "Rich Text Editor, wysiwygarea, Editor, wysiwygarea frame, Editor, wysiwygarea document editable"
```

Comment:
Hmm, did you test NVDA with Chrome? I can see that in Chrome it's read the way you described but in Firefox – the way I described.

It seems that it would be better to annotate browsers used for each screen reader.